containers/test-apps/courier/pom.xml

```diff
@@ -6,13 +6,13 @@
 
     <groupId>twosigma</groupId>
     <artifactId>courier</artifactId>
-    <version>1.5.18</version>
+    <version>1.5.19</version>
 
     <name>courier</name>
     <url>https://github.com/twosigma/waiter/tree/master/test-apps/courier</url>
 
     <properties>
-        <grpc.version>1.30.2</grpc.version>
```

Comment:
We use the client in Waiter for integration tests and there a newer grpc version leads to class loading issues due to transitive dependencies being of different versions.